src/java.base/share/classes/javax/crypto/spec/DESKeySpec.java

```diff
@@ -41,12 +41,12 @@ public class DESKeySpec implements java.security.spec.KeySpec {
      */
     public static final int DES_KEY_LEN = 8;
 
-    private byte[] key;
+    private final byte[] key;
 
     /*
      * Weak/semi-weak keys copied from FIPS 74.
      *
-     * "...The first 6 keys have duals different than themselves, hence
+     * "...The first 6 keys have duals different from themselves, hence
```

Comment:
Suggest you leave this alone as this is a direct quote from FIPS 74 (page 10).

Reply:
Keeping the original.